main.py

```diff
@@ -64,3 +65,21 @@ def handle_service_exception(
     return JSONResponse(
         status_code=status.HTTP_400_BAD_REQUEST, content={"detail": str(exception)}
     )
+
+
+@app.exception_handler(AuthenticationException)
+def handle_authentication_exception(
+    request: Request, exception: AuthenticationException
+) -> JSONResponse:
+    return JSONResponse(
+        status_code=status.HTTP_400_BAD_REQUEST, content={"detail": str(exception)}
+    )
```

Comment:
401 code instead of 400 as this exception needs to inform about authentication failure